src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -98,6 +99,12 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
     }
   }, [shouldOpenPaymentModal, isAuthenticationModalOpen]);
 
+  useEffect(() => {
+    if (venue?.showZendesk) {
+      showZendeskWidget();
+    }
+  }, [venue]);
+
```

Comment:
As above: Should this have an `else` clause where it calls `hideZendeskWidget`?

Also, since this code snippet is repeated in a number of files, there is probably a more DRY way we can implement the pattern. Either by making the check in a 'higher' component in the tree if possible, or at the very least by wrapping up this logic as a custom hook.

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/103